src/data/padics/hensel.lean

```diff
@@ -163,12 +163,14 @@ have hdzne' : (↑(F.derivative.eval z) : ℚ_[p]) ≠ 0, from
   λ h, hdzne $ subtype.ext_iff_val.2 h,
 let ⟨q, hq⟩ := F.binom_expansion z (-z1) in
 have ∥(↑(F.derivative.eval z) * (↑(F.eval z) / ↑(F.derivative.eval z)) : ℚ_[p])∥ ≤ 1,
-  by {rw padic_norm_e.mul, apply mul_le_one, apply padic_int.norm_le_one, apply norm_nonneg, apply h1},
+  by { rw padic_norm_e.mul, exact mul_le_one (padic_int.norm_le_one _) (norm_nonneg _) h1 },
```

Comment:
There are some non-newline changes in the PR, I'm assuming this is intentional.

Reply:
They're very minor (like squashing `apply`s into `exact` here or using `sum_erase` in `caratheodory`).

Reply:
Looks like our reviews crossed; I thought these were all OK.